packages/upload-api/src/blob/lib.js

```diff
@@ -0,0 +1,85 @@
+import { Failure } from '@ucanto/server'
+
+export const BlobItemNotFoundName = 'BlobItemNotFound'
+export class BlobItemNotFound extends Failure {
+  /**
+   * @param {import('@ucanto/interface').DID} [space]
+   */
+  constructor(space) {
+    super()
+    this.space = space
+  }
+
+  get name() {
+    return BlobItemNotFoundName
+  }
+
+  describe() {
+    if (this.space) {
+      return `Blob not found in ${this.space}`
+    }
+    return `Blob not found`
+  }
+
+  toJSON() {
+    return {
+      ...super.toJSON(),
+      space: this.space,
+    }
+  }
+}
+
+export const BlobSizeOutsideOfSupportedRangeName =
+  'BlobSizeOutsideOfSupportedRange'
+export class BlobSizeOutsideOfSupportedRange extends Failure {
+  /**
+   * @param {Number} maxUploadSize
+   */
+  constructor(maxUploadSize) {
+    super()
+    this.maxUploadSize = maxUploadSize
+  }
+
+  get name() {
+    return BlobSizeOutsideOfSupportedRangeName
```

Comment:
```suggestion
    return 'BlobSizeOutsideOfSupportedRangeName'
```

Reply:
this is the name variable... I like to use this as a const so that I easily can check in the tests